src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,6 +24,193 @@ const formConfig = {
   title: 'Apply for education benefits',
```

Comment:
Like in pensions, could it be helpful to specify "Veteran education benefits"? Not sure when/if the specification is helpful.